src/main/java/teammates/ui/controller/InstructorSearchPageData.java

```diff
@@ -21,10 +41,71 @@
     public boolean isSearchCommentForStudents;
```

Comment:
Ah, I'm not specifically referring to the `isSearchCommentForStudents` boolean, I meant all of the public variables above here.